### DISCUSSION TITLE
Questions about 01.moel deviation

### DISCUSSION THREAD

**Discussion by Edinbur** [deepmodeling/dpgen#1514] [Q&A]
Can the model_devi_dt parameter in param.json in dpgen run be set to 1? Because I found that when I set it to 1, I get an error when I do 01.model deviation, but not when I set it to 0.001!

**Comment by njzjz** [comment]
The unit of `model_devi_dt` is ps

**Comment by Edinbur** [comment]
这是来自QQ邮箱的假期自动回复邮件。
 
您好，我最近正在休假中，无法亲自回复您的邮件。我将在假期结束后，尽快给您回复。
